### PR TITLE
Fetch TeamStack from API

### PR DESCRIPTION
**Pull request by Ase020**
Pull data from `/stack` endpoint.

### CODE REVIEW

File: src/APP/pages/products2/data.js [hunk 1]
```diff
@@ -1,284 +1,284 @@
...
+//       name: "Fred Ouko",
+//       title: "Product Designer",
+//       headshot: fred,
+//       portfolio: "https://www.linkedin.com/in/fred-ouko-918293bb",
+//     },
+//     {
+//       id: 4,
+//       name: "Aaron Abubakar",
+//       title: "Product Designer",
+//       headshot: aaron,
```
Comment (JimmyTron) [inline src/APP/pages/products2/data.js:130]: just delete these lines let version control do its work

**Review by JimmyTron — CHANGES_REQUESTED** [review]
Remove commented code

File: src/APP/pages/products2/data.js (hunk 1 above)
Comment (Ase020) [reply src/APP/pages/products2/data.js:130]: I left the comments so that can you get some of the headshots not yet uploaded on the backend.

**Review by collinskasyoki — CHANGES_REQUESTED** [review]
The commented out code is unecessary. One can always go back to Version Control to view the data as @JimmyTron has suggested.

**Review by collinskasyoki — APPROVED** [review]

**Review by JimmyTron — APPROVED** [review]

**Pull request merged**